.github/workflows/publish-dev-container.yml

```diff
@@ -0,0 +1,48 @@
+
+name: Release, Build, and Push
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+-**
+    branches:
+      - main
+
+jobs:
+  build-dev-image:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: hub.opensciencegrid.org/macrostrat/tileserver
```

Comment:
Do we mean to be grabbing things from the macrostrate container, or is this a copy-paste error?